lib/src/property.dart

```diff
@@ -652,17 +657,15 @@ class Hsla implements _StyleProperty, ColorBase {
     var b = value.toInt() & 0xff;
 
     // Convert alpha to 0..1 from (0..255).
-    if (a != null) {
```

Comment:
🤕

Reply:
I do wonder if this was meant to be something like `a != 0`? The initializing expression can never be `null` can it?

Reply:
No it couldn't, this was just nonsense code lol. Sorry probalby shouldn't have commented at all lol :D